aclpub2/templates/proceedings.tex

```diff
@@ -102,7 +102,7 @@
 {\large
 
 \noindent
-\textcopyright \VAR{conference.start_date.year} The Association for Computational Linguistics\\
+\textcopyright \VAR{conference.start_date.year} \VAR{conference.publisher}\\
```

Comment:
@crux82 do we want to use the `conference.publisher` value here on the copyright page, or just stick with "The Association for Computational Linguistics"?

Reply:
Good idea ti replace with the publisher!